src/transitions/Slide.js

```diff
@@ -82,14 +85,17 @@ export default class Slide extends Component {
   handleEntering = (element) => {
     const { transitions } = this.context.theme;
     element.style.transition = transitions.create('transform',
-      `${this.props.transitionDuration}ms`);
+      `${this.props.enterTransitionDuration}ms`, undefined, easing.easeOut);
```

Comment:
So, i am thinking of something this:
```
transitions.create({
      properties: 'transform', // String/Array
      duration: transitions.durations.enteringScreen, // Number - nobody will use anything else than ms anyway
      delay: ... //as above
      easing: transitions.easing.easeOut, // String, probably some check if it is from standard functions?
    });
```
Where that easing/duration constants would be best to obtain from context theme, since i suspect it  would be appealing to change timings according to user agent (mobile/pc, ...)